proteus/config/__init__.py

```diff
@@ -0,0 +1,6 @@
+import platform
+
+if platform.node().startswith('garnet') or platform.node().startswith('copper'):
```

Comment:
Unfortunately, this doesn't appear to trigger on the Garnet compute nodes.  I may need to do a quick interactive debug session to see what's happening on the back-end here.